spec/presenters/redirect_versions_presenter_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe RedirectVersionsPresenter do
+  subject(:last_updated_by) { described_class.new(redirect_version).last_updated_by }
```

Comment:
Nice done. But this subject belongs to top to line 5, since the subject on line 4 in the describe block.

e.g

```ruby
  describe '#last updated by' do
    subject(:last_updated_by) { described_class.new(redirect_version).last_updated_by }
    #...
  end
```

Don't forget to append a '#' on line 4 too.